distro/wildfly/modules/src/main/modules/com/h2database/h2/main/module.xml

```diff
@@ -24,12 +24,20 @@
 
 <module xmlns="urn:jboss:module:1.3" name="com.h2database.h2">
 
-    <resources>
-        <resource-root path="h2-@version.h2@.jar"/>
-    </resources>
-    <dependencies>
-        <module name="javax.api"/>
-        <module name="javax.transaction.api"/>
-        <module name="javax.servlet.api" optional="true"/>
-    </dependencies>
+  <resources>
+    <resource-root path="h2-@version.h2@.jar"/>
+  </resources>
+  <dependencies>
+    <module name="java.base"/>
+    <module name="java.compiler"/>
+    <module name="java.desktop"/>
```

Comment:
❓Does it really need `java.desktop`?
Update: I saw that the official wildfly repo defines it as well.

Reply:
Yes, also the commit message here (https://github.com/camunda/camunda-bpm-platform/pull/4135/commits/ff92ca47d6e318bae7682a0b5a1431e4b419840c) contains the result of the jdeps command